packages/organisations-api/get-organisation-disruptions/index.ts

```diff
@@ -0,0 +1,125 @@
+import { Service } from "@create-disruptions-data/shared-ts/disruptionTypes";
+import { Datasource } from "@create-disruptions-data/shared-ts/enums";
+import { notEmpty } from "@create-disruptions-data/shared-ts/utils";
+import { getActiveDisruptions } from "@create-disruptions-data/shared-ts/utils/dynamo";
+import { fetchService } from "@create-disruptions-data/shared-ts/utils/refDataApi";
+import { APIGatewayEvent } from "aws-lambda";
+import * as logger from "lambda-log";
+import { randomUUID } from "crypto";
+
+const getServiceCentrePoint = async (service: Service) => {
+    const serviceInfo = await fetchService(
+        service.dataSource === Datasource.bods ? service.lineId : service.serviceCode,
+        service.nocCode,
+        service.dataSource,
+        logger,
+    );
+    return [serviceInfo?.centrePointLat, serviceInfo?.centrePointLon];
+};
+
+const getOrganisationDisruptions = async (orgId: string) => {
+    try {
+        const disruptionsTableName = process.env.DISRUPTIONS_TABLE_NAME as string;
+        const activeDisruptions = await getActiveDisruptions(disruptionsTableName, logger, orgId);
+
+        const disruptionsFormattedForMap = await Promise.all(
+            activeDisruptions.flatMap(async (disruption) => {
+                if (!disruption.consequences || disruption.consequences.length === 0) {
```

Comment:
do we need to check disruption.consequences exists before doing disruption.consequences.length

Reply:
i don't believe so